src/pages/partners/payment-agent/_faq-data.tsx

```diff
@@ -7,6 +7,11 @@ import { DerivStore } from 'store'
 
 const TextLink = styled(LinkText).attrs({ as: 'span' })``
 
+type StoreDataType = {
+    is_livechat_interactive: boolean
+    LC_API: any
+    setFirstLoadOpenLc: any
+}
```

Comment:
Would we still need this if the original store file is typed? And if so, can we export the type from the store?

Reply:
@Sean Thanks for pointing. I have used DerivStoreType instead.